modulus/distributed/utils.py

```diff
@@ -130,90 +130,91 @@ def gather_loss(loss: float, dst_rank: int = 0, mean: bool = True):
         return None
 
 
-# # distributed primitives
-# def _transpose(tensor, dim0, dim1, group=None, async_op=False):
-#     # get input format
-#     input_format = get_memory_format(tensor)
+# distributed primitives
+def _transpose(tensor, dim0, dim1, group=None, async_op=False):
```

Comment:
I'd prefer something like `distributed_transpose` or similar with a non-private naming to make this more specific

Reply:
Made this change